contracts/RelayHub.sol

```diff
@@ -205,6 +205,9 @@ contract RelayHub is IRelayHub {
         vars.success = success;
         vars.innerGasUsed = vars.gasBeforeInner-gasleft();
         (vars.status, vars.relayedCallReturnValue) = abi.decode(relayCallStatus, (RelayCallStatus, bytes));
```

Comment:
Can it be a local variable now?

Reply:
nope. adding a single stack variable  (even unused) before this line breaks it..